gnucash/gnome/gnc-budget-view.c

```diff
@@ -907,9 +907,9 @@ budget_col_source(Account *account, GtkTreeViewColumn *col,
             xaccSPrintAmount(amtbuff, numeric,
                              gnc_account_print_info(account, FALSE));
             if (gnc_is_dark_theme (&color))
-                g_object_set(cell, "foreground", "darkgray", NULL);
+                g_object_set(cell, "foreground",  gnc_numeric_negative_p(numeric) ? "darkred" :"darkgray", NULL);
```

Comment:
There is a preference setting for this (Edit->Preferences->General->Disply negative numbers in red). Can you make this behaviour obey that preference setting ?